src/python/py/models/builder.py

```diff
@@ -291,6 +291,9 @@ def make_genai_config(self, model_name_or_path, extra_kwargs, out_dir):
                 "pad_token_id": config.pad_token_id if hasattr(config, "pad_token_id") and config.pad_token_id is not None else config.eos_token_id[0] if isinstance(config.eos_token_id, list) else config.eos_token_id,
                 "type": self.model_type[ : self.model_type.find("For")].lower(),
                 "vocab_size": self.vocab_size,
+                "model_name": model_name_or_path,
+                "precision": precision,
```

Comment:
Can you add `precision` into the dictionary so that the alphabetical order remains?